packages/backend/src/income/handler/getIncomeHandler.ts

```diff
@@ -0,0 +1,18 @@
+import type { Context } from 'koa'
+import { getIncomeService } from '../service/getIncomeService'
+import { validateGetIncomeInput } from '../validation'
+import { errorStatusMapper } from '../../utilities/errorStatusMapper'
+import { STATUS_SUCCESS_200 } from '../../models/statusCodes'
+
+export async function getIncomeHandler(ctx: Context) {
+  try {
+    const input = ctx.request.body
+    validateGetIncomeInput(input)
+    const income = await getIncomeService(input)
+    ctx.status = STATUS_SUCCESS_200
+    ctx.body = { income }
```

Comment:
should be ctx.body = {incomes}. Change the variable name on line 11 and update the unit test to match